app/views/wishlists/index.html.erb

```diff
@@ -4,6 +4,7 @@
     <% @wishlists.each do |wishlist| %>
       <li>
         <%= link_to wishlist.name, "#" -%>
+        <%= button_to "Delete", wishlist_path(wishlist.id), method: :delete -%>
```

Comment:
you can replace `wishlist_path(wishlist.id)` with `wishlist`

for rails it's enough